index.html

```diff
@@ -387,6 +388,45 @@ <h3> <span class="glyphicon glyphicon-exclamation-sign"></span> Legal and Privac
  </div>
 </section>
 
+<section>  
+  <div class="content" id="cvrm">
+    <h1>Predictive Policing</h1>
+    <p>Since 2012, the CPD has used a statistical model to predict which individuals are most likely to be involved in shooting incidents. The model, developed in collaboration with the Illinois Institute of Technology, was initially called the Strategic Subject List (SSL) and was later revised and renamed the Crime and Victimization Risk Model (CVRM). This model was quietly <a id="cvrmlink" href="https://igchicago.org/2020/01/23/advisory-concerning-the-chicago-police-departments-predictive-risk-models/">decommissioned</a> by the CPD in 2019. </p>
+    <div class="row">
+    	<div class="col-md-6"></div>
+        <div class="col-md-6"><img src="img/CVRMimage.png" alt="Socmint" class="img-rounded"></div>
+    </div>
+ </div>
+</section>
+
+<section class="nonheading">  
+  <div class="content">
+  	<div class="row">
+    <h3> <span class="glyphicons glyphicon glyphicon-map-marker"></span> Where</h3> 
+    <p>The CVRM was meant to play a key role in CPD’s new <a href="https://www.rand.org/pubs/research_reports/RR3242.html">Strategic Decision and Support Centers (SDSCs)</a>, real time crime centers which are now located in all but two of Chicago’s 22 districts. Officers could also view at risk individuals identified by the CVRM on a phone app.</p>
+
+    <h3> <span class="glyphicon glyphicon-piggy-bank"></span> Costs</h3> 
+    <p>CPD received $3.8 million in federal grants to develop the SSL and CVRM.</p>
+    </div>
+	<div class="row">
+		<div class="col-md-6"></div>
+		<div class="col-md-6"><img src="img/SSL-Dash.jpg" style="width:100%"><h5>A screenshot of an individual's risk classification produced by the SSL. Obtained via FOIA request by <a href="https://southsideweekly.com/predictive-policing-long-road-transparency/">South Side Weekly</a>.</div>
+	</div>
+  	<div class="row">
+    <h3> <span class="glyphicon glyphicon-eye-open"></span> Capabilities </h3>
+    <p> The CVRM outputs a risk classification for anyone arrested in Chicago over the past four years (over 300,000 people). The model’s output is based on risk factors such as a subject’s age and involvement in various types of crimes, with more recent incidents being given a higher weight than older ones. Individuals’ scores are also influenced by the scores of co-arestees. IIT provided a <a href="https://home.chicagopolice.org/wp-content/uploads/2019/01/FACT-SHEET-Crime-and-Victimization-Risk-Model-1.pdf">fact sheet</a> which describes the model in greater detail.</p>
```

Comment:
This is not technically true. "The CVRM outputs a risk classification for anyone arrested in Chicago over the past four years (over 300,000 people)." The part about arrests is true but that's not the only people placed on the list. If you were the _victim_ of a violent crime, you were also put on the SSL. Also please include this statistic "56 percent of black men in the city ages 20 to 29 have an SSL score " 

```
The newest data release about the SSL showed that nearly 400,000 people were assigned scores on the list, using primarily eight variables: age, the number of times an individual has been shot, the number of times they were the victim of assault or battery, gang affiliation, arrests for violent offenses, drugs, or weapons, and whether someone is getting arrested more frequently over time.
```
https://www.chicagomag.com/city-life/August-2017/Chicago-Police-Strategic-Subject-List/

Reply:
I would also strongly caution about using IIT's "risk model 1" the SSL evolved over time and there are a number of risk models. It was supposed to be targeted to the "most likely to be violent" and then gradually expanded to included 400,000 names. The RAND corp study you mention later only studied v1 of the algorithm, which CPD disputes because they changed the model over time. (I think by the time it was decommissioned it was on version 7)

Reply:
CPD gave everyone with an arrest an SSL score but never purged it if charges where dropped.

Reply:
I actually think we're talking about two different RAND studies: (1) http://link.springer.com/article/10.1007/s11292-016-9272-0 from 2016, about v1, and (2) https://www.rand.org/pubs/research_reports/RR3242.html from 2019, about SDSCs in general, which talks about the CVRM so at least v5 or v6